src/CONST.ts

```diff
@@ -3306,6 +3306,7 @@ const CONST = {
 
     CONCIERGE_TRAVEL_URL: 'https://community.expensify.com/discussion/7066/introducing-concierge-travel',
     BOOK_TRAVEL_DEMO_URL: 'https://calendly.com/d/ck2z-xsh-q97/expensify-travel-demo-travel-page',
+    TRIP_ID_URL: (tripID: string, isProduction: boolean) => (isProduction ? `https://travel.expensify.com/trips/${tripID}` : `https://staging.travel.expensify.com/trips/${tripID}`),
```

Comment:
should we add the beginning of these URLs to CONST as well? Similar to     ```TRAVEL_TERMS_URL: `${USE_EXPENSIFY_URL}/travelterms`,```

Reply:
given that we'll have more travel urls, that is a fair ask.